### DISCUSSION TITLE
How to send full query, ouput and status text to file or pager?

### DISCUSSION THREAD

**Discussion by GlassGruber** [dbcli/mycli#1107] [Q&A]
Hei there!
I've found this https://github.com/dbcli/mycli/issues/455 , but I'd like to do the exact opposite! 🙃 

So I'd like to send:
- the full query
- the full result output 
- the status text

Is it possible? I'm not interested in doing this for all queries, is there a one off command to do this?

Example, the below text should all be sent to a file or pager
```text
SELECT special_column FROM my_table

+--------------------+
| special_column     |
+--------------------+
| some row of data 1 | 
| some row of data 2 | 
| some row of data 3 | 
| some row of data 4 | 
    -- ETC ... --
+--------------------+

X rows in set
Time: 0.347s
```

I take the chance to say thank you all for this very nice tool! Hats to you 🎩👌

**Comment by GlassGruber** [comment]
Hey I understand now that with `tee` command I can get most of what I need, the query and the result are sent, also the prompt which can be a nice extra though a bit verbose, but not the status text with the number of rows.  
I thought `tee` was like `once` command that sends only the result to a file (isn't this somewhat inconsistent?).

So I guess I've found a sort of "good enough" solution for now, still a bit inconvenient to use `tee` for just a command and `notee` right after.

**Comment by rolandwalker** [comment]
Yes, I agree that https://github.com/laixintao/iredis has a better solution, allowing trailing `|` through shell commands.  Then you don't have to set something and unset it again.
